serial_communication/web/app.py

```diff
@@ -92,18 +101,43 @@ def send_auth_route():
 
 def update_schedule():
     while True:
-        schedule.run_pending()
-        time.sleep(1)
+        schedule.run_pending() # run background task after specified delay
+        time.sleep(10)
 
-# def delayed_execution():
-#     # Delay for 5 seconds
-#     time.sleep(20)
-#     # Call the function after the delay
-#     send_ngrok_link()
+def one_time_task():
+    try:
+        global boot_up_message_send
+        # Delay the execution of send_ngrok_link() by 5 minutes
+        
+        # Check if send_ngrok_link() has not been called yet
+        if not is_ngrok_link_sent():
+            time.sleep(20)
+            if not boot_up_message_send:
+                send_message("Orange Pi just boot-up time stamp: "+ fetch_current_time_online())
```

Comment:
I think it might not send boot up message @mhamidjamil please make sure it does.